### ISSUE TITLE
Runtime errors in elif statements report the line number of the if statement instead of its own

### ISSUE THREAD

**Issue by goatchurchprime** [godotengine/godot#40844]
**Godot version:**
3.2.2.rc1

**OS/device including version:**
Windows 10

**Issue description:**
If you make a series of `if-elif-elif` blocks in GDScript and there is a error in the condition statement in one of the `elif`s, then the error message incorrectly reports that it was on the line number of the `if`-statement.  

This makes debugging very difficult when you don't know about the issue and you have some very long if-blocks.  I spent a long time scratching my head at the perfectly good line nowhere near where the error actually occurred.

**Steps to reproduce:**

Consider the code below, which prints the correct error on the correct line:

> **res://immediate_code.gd:41 - Division By Zero in operator '/'.**

```GDscript
19  func _run():
20	var i = 10
21	var z = 0
22	
23	#i = 11  # uncomment this line to create the errors
24	
25	if i == 20:
26		print("i is twenty")
27	elif i == 10:
28		print("i is ten")
29	
30	# Error 1	
31	#elif i == "three":
32	#	print("garbage in garbage out")
33	
34	# Error 2	
35	#elif i == 1/z:
36	#	print("more garbage")
37	
38	# Error 3	
39	elif len(i) == 3:
40		print("more garbage")
41	print("reached the end", 1/z)
```
When I uncomment the `i=11` on line 23 and run the error is:

> **res://immediate_code.gd:25 - Error calling built-in function 'len': Object can't provide a length.**

When I uncomment lines 35 and 36, the error is:

> **res://immediate_code.gd:25 - Division By Zero in operator '/'.**

When I uncomment lines 31 and 32, the error is:

> ** res://immediate_code.gd:25 - Invalid operands 'int' and 'String' in operator '=='.**

The reason for testing with all these errors is that at first it looked like a problem with type-checking.  But this proves that it's every run-time error in the elif-statement that goes wrong.  Parser errors are okay: for example `elif i == nonexistingfunction(10):` on line 27 gives the correct error line.

**Minimal reproduction project:**
N/A

**Comment by akien-mga** [comment]
> Godot version:
3.2.2.rc1

Time to update :)
This is fixed in 3.2.2-stable already.

**Comment by goatchurchprime** [comment]
goddammit.  I can't keep up with this place.  I'll catch you out one day!
